libsentrykube/service.py

```diff
@@ -210,10 +210,7 @@ def get_hierarchical_value_overrides(
 
         try:
             service_override_file = (
-                get_service_value_override_path(
-                    service_name, override_group.name, external
```

Comment:
This `external` param is used by some deployments (vector-aa, anti-abuse-pop, frontend) so removing this will break their usage for this lookup.

Reply:
I think `service_regions_path` should already handle this because it considers the `external` param a few lines above

Reply:
Ah yes, good call sorry.